docs/fp_primer.md

```diff
@@ -253,8 +260,9 @@ There are 5 different ways that accuracy requirements are defined in the spec:
    The solution to the issue of not scaling with precision of floating point is
    to use units of least precision.
 
-   There are many more formal definitions of ULP, but at a high level it is the
-   difference from a specific value to the next nearest f32 value.
+   ULP(X) is min (b-a) over all pairs (a,b) of representable floating point 
+   numbers such that (a <= X <= b). For a more formal discussion of ULP see 
```

Comment:
Sorry, a nit here:  I goofed and we also must have a not equal to b.
It's written correctly in the WGSL spec.

Reply:
Done